cma/evolution_strategy.py

```diff
@@ -4646,4 +4661,30 @@ def update(es):
     es.objective_function_complements = [_al]
     es.augmented_lagrangian = _al
     es.best_feasible = best_feasible_solution
+
+    if post_optimization:
+        positive_constraints = np.where(np.array(g(es.result.xfavorite)) > 0)
+        if len(positive_constraints[0]) > 0:
+            x_post_opt, es_post_opt = fmin_con(lambda x: sum([gi**2 if gi > 0 else 0 for gi in g(x)]),
+                                               es.result.xfavorite, sigma0 / 1000, g=g, h=h, **kwargs)
+            if es_post_opt.best_feasible.info is not None:
+                f_x_post_opt = objective_function(es_post_opt.best_feasible.info["x"])
+
+                best_feasible_solution_post_opt = ot.BestSolution2()
+                best_feasible_solution_post_opt.update(
```

Comment:
it looks like it would be better to use a copy of ```es_post_opt.best_feasible.info``` and update/add the missing entries

Reply:
I updated the code to use a copy of `es_post_opt.best_feasible.info` and update `post_opt_info['f']`.